.gitignore

```diff
@@ -30,8 +30,9 @@ _testmain.go
 .coverprofile
 /gover.coverprofile
 e2e-tests
-
+coverage.out
```

Comment:
I'm happy to see this line being added. coverage files clutter my index all the time :+1: